js/phemex.js

```diff
@@ -925,7 +925,11 @@ module.exports = class phemex extends Exchange {
             'symbol': market['id'],
             // 'id': 123456789, // optional request id
         };
-        const response = await this.v1GetMdOrderbook (this.extend (request, params));
+        let method = 'v1GetMdOrderbook';
+        if (market['linear']) {
```

Comment:
@AlgoQ, we can't do it like this, `BTC/USD:USD´ is also a linear market but uses V1 instead
```
n phemex fetchOrderBook "BTC/USD:USD"          
2023-02-06T16:36:30.593Z
Node.js: v17.9.0
CCXT v2.7.14
phemex.fetchOrderBook (BTC/USD:USD)
BadRequest phemex {"error":{"code":6001,"message":"invalid argument"},"id":null,"result":null}

---------------------------------------------------
[BadRequest] phemex {"error":{"code":6001,"message":"invalid argument"},"id":null,"result":null}

    at throwExactlyMatchedException  js/base/Exchange.js:2419              throw new exact[string] (message);                                              
    at handleErrors                  js/phemex.js:3734                     this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, feedbac…
    at                               js/base/Exchange.js:603               const skipFurtherErrorHandling = this.handleErrors (response.status, response.s…
    at processTicksAndRejections     node:internal/process/task_queues:96                                                                                  
    at async timeout                 js/base/functions/time.js:181         return await Promise.race ([promise, expires.then (() => { throw new TimedOut (…
    at fetch2                        js/base/Exchange.js:2042              return await this.fetch (request['url'], request['method'], request['headers'],…
    at request                       js/base/Exchange.js:2046              return await this.fetch2 (path, api, method, params, headers, body, config, con…
    at fetchOrderBook                js/phemex.js:932                      const response = await this[method] (this.extend (request, params));            
    at async run                     examples/js/cli.js:293                const result = await exchange[methodName] (... args)                            

phemex {"error":{"code":6001,"message":"invalid argument"},"id":null,"result":null}
```